apis/v1alpha3/conversion.go

```diff
@@ -23,12 +23,14 @@ import (
 )
 
 // Convert_v1beta1_VirtualMachineCloneSpec_To_v1alpha3_VirtualMachineCloneSpec is an autogenerated conversion function.
```

Comment:
(just wanted to place it as a comment somewhere)

Please update the PR description to reflect the new state of the implementation

Reply:
Sure. Thanks for the reminder.